### PR TITLE
Addressing issues 162, 161, 160, 159, 158, 157, 156, 155, 154, 143, 116, 69

### PR DESCRIPTION
**Pull request by kheki4**

### CODE REVIEW

File: src/camTempMonitor.h [hunk 1]
```diff
@@ -27,14 +27,22 @@ Q_DECLARE_METATYPE(std::vector<double>)
     thus a recEventTracker instance also must exist
 */
 
+/** The temperature that we deem as default value when cannot be measured, and the minimum that can be measured */
+static const double MINIMUM_DEVICE_TEMPERATURE = -1.0;
```
Comment (sasfog) [inline src/camTempMonitor.h:31]: This should not be left as a global variable

File: src/subwindows/remoteCCDialog.cpp [hunk 2]
```diff
@@ -149,9 +149,9 @@ void RemoteCCDialog::createForm() {
     mainLayout->addWidget(comGroup);
 
     connect(connectUDPButton, SIGNAL(clicked()), this, SLOT(onConnectUDPClick()));
-    connect(disconnectUDPButton, SIGNAL(clicked()), this, SLOT(onDisconnectUDPClick()));
+    connect(disconnectUDPButton, SIGNAL(clicked()), this, SLOT(disconnectUDP()));
```
Comment (sasfog) [inline src/subwindows/remoteCCDialog.cpp:152]: These should have the on prefix as they are reacting to signals

File: src/subwindows/streamingSettingsDialog.cpp [hunk 3]
```diff
@@ -184,9 +184,9 @@ void StreamingSettingsDialog::createForm() {
     mainLayout->addWidget(comGroup);
 
     connect(connectUDPButton, SIGNAL(clicked()), this, SLOT(onConnectUDPClick()));
-    connect(disconnectUDPButton, SIGNAL(clicked()), this, SLOT(onDisconnectUDPClick()));
+    connect(disconnectUDPButton, SIGNAL(clicked()), this, SLOT(disconnectUDP()));
```
Comment (sasfog) [inline src/subwindows/streamingSettingsDialog.cpp:187]: These should have the on prefix as well

**Review by sasfog — CHANGES_REQUESTED** [review]
Please make the changes based on the review

File: src/camTempMonitor.h (hunk 1 above)
Comment (kheki4) [reply src/camTempMonitor.h:31]: Made it inside the class, but could not access it elsewhere, left it like this with a comment

**Review by sasfog — APPROVED** [review]

**Pull request merged**